client/containers/DashboardSettings/PubSettings/Doi.js

```diff
@@ -88,21 +180,28 @@ class Doi extends Component {
 	}
 
 	render() {
-		const {
-			pubData: { doi },
-			canIssueDoi,
-		} = this.props;
+		const { pubData, canIssueDoi } = this.props;
 		const { justSetDoi } = this.state;
+		const hasExistingDeposit = justSetDoi || Boolean(pubData.crossrefDepositRecordId);
 
 		return (
 			<div className="pub-settings-container_doi-component">
 				{this.renderStatusMessage()}
 				{this.renderCollectionContextMessage()}
 				{this.renderDoi()}
 
+				{!hasExistingDeposit && (
```

Comment:
I think there needs to be a separate bit of copy for non-admins, who get the rundown on what the button does but are unable to click it, as seen here:

![image](https://user-images.githubusercontent.com/2208769/88571615-eeedc900-d00b-11ea-8cee-eeeb0da408a0.png)

For those users, who do not have any write access to the DOI, it may be simplest to have a different render branch that shows them the DOI if it exists, and either the "admins may assign a DOI" copy or nothing at all otherwise.